src/mobile/src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { NavigationContainer } from '@react-navigation/native';
+
+import { registerRootComponent } from 'expo';
+import Navigation from './navigation/Navigation';
```

Comment:
Remove extra space, and put a space between our import and dependency import
```suggestion
import { NavigationContainer } from '@react-navigation/native';
import { registerRootComponent } from 'expo';

import Navigation from './navigation/Navigation';
```